config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -39,6 +39,55 @@ periodics:
         requests:
           cpu: 6
           memory: 6Gi
+- name: ci-kubernetes-node-kubelet-kubetest2
+  cluster: k8s-infra-prow-build
+  interval: 1h
+  decorate: true
+  path_alias: k8s.io/kubernetes
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  decoration_config:
+    timeout: 65m
+  annotations:
+    fork-per-release: "true"
+    fork-per-release-periodic-interval: 1h 2h 6h 24h
+    testgrid-dashboards: sig-release-master-blocking, sig-node-release-blocking
+    testgrid-tab-name: node-kubelet-master-kubetest2
+    testgrid-alert-email: kubernetes-sig-node+testgrid@googlegroups.com,kubernetes-sig-node-test-failures@googlegroups.com
```

Comment:
perhaps we can start without e-mail notifications?

Reply:
seems like tests need this alert to get passed. 

` Must have alert_options` ref: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/24308/pull-test-infra-unit-test/1459117935019692032

Reply:
sure, can we please only keep `kubernetes-sig-node-test-failures@googlegroups.com`, thank you